src/obsidian_ai_hub/coding/service.py

```
@@ -210,6 +213,65 @@ async def run_coding_turn_stream(
                             error=event.get("error"),
                         )
                         yield f"data: {json.dumps({'event': 'orchestrator_tool_call_end', 'call_id': event['call_id'], 'call_key': event['call_key'], 'tool_name': event['tool_name'], 'status': event['status'], 'result': event['result'], 'error': event.get('error'), 'phase': phase, 'phase_turn': phase_turn, 'iteration': event['iteration'], 'call_index': event['call_index']}, ensure_ascii=False)}\n\n"
+                    elif evt_type == "user_question":
+                        ask_call = event.get("ask_call", {})
+                        questions_data = event.get("questions", [])
+                        hitl_run_id = f"hitl_ask_{uuid.uuid4().hex[:12]}"
```

Ruff (0.16.3)

[error] Undefined name `uuid`

(F821)